src/patternfly/components/Check/styles.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../patternfly-utilities";
+
+.pf-c-check {
```

Comment:
Should `pf-c-checkbox` and `pf-c-radio` be applied to the individual elements? I see `check` as a wrapper.

Reply:
Ya I agree. That is likely what will happen once we finalize the design for these. I followed what Bootstrap did for the time being but they don't customize them individually.